pkg/minishift/openshift/version/openshift_versions.go

```diff
@@ -153,3 +142,47 @@ func getGithubReleases() ([]string, error) {
 	}
 	return releaseTags, nil
 }
+
+func GetSortedOpenShiftTags(minSupportedVersion, defaultVersion string) ([]string, error) {
+	var releaseList []string
+
+	data, err := getGithubReleases()
+	if err != nil {
+		return []string{}, err
+	}
+
+	for _, releaseTag := range data {
+		if valid, _ := IsGreaterOrEqualToBaseVersion(releaseTag, minSupportedVersion); valid {
+			if valid, _ := IsGreaterOrEqualToBaseVersion(releaseTag, defaultVersion); valid {
+				releaseList = append(releaseList, releaseTag)
+			} else {
+				if !isPrerelease(releaseTag) {
+					releaseList = append(releaseList, releaseTag)
+				}
+			}
+		}
+	}
+
```

Comment:
I know that you extracted this because you need the release list to map the latest version but any specific reason for this `sortToSemVerTags` does that was not working before as it should be? Also you need to extract that for downstream also since the code is part of upstream only.

Reply:
@praveenkumar String sort is not working and hence need to do `semver.Sort(versionTags)`.
When I do string sort, I am getting `v3.9.0` at the bottom instead of `v3.10.0-rc.1`.
Hence, I am converting string versions into semver tag and returning the string format only.
Hope it will help.

Reply:
I would even argue if we really should do `getGithubReleases()` is this function, as it prevents you from doing easy unittests on this function. If we can provide the results to `getOpenShiftTags` we can easily write a test against this.

Reply:
Change to "getGithubReleases" is out of scope so we prefer to create another task as refactoring task.

Reply:
Done @gbraad :smile: